src/store/reviewSpot/index.js

```diff
@@ -217,8 +224,19 @@ const actions = {
             return;
         }
         const response = await mayaClient.post(`/owner/spot-update?spot-id=${state.SO.spotId}`);
-        commit('set-global-error', 'Your request was registered successfully');
-        return response.data;
+        if (response.DisplayMsg) {
+            if (response.ErrorCode === 6) {
+                // Conflict error (already exists)
+                commit('set-global-error', 'This request has already been submitted.');
```

Comment:
I don't see the need of checking the ErrorCode, as DisplayMsg should have the proper msg.